app/uk/gov/hmrc/transitmovementsrouter/models/sdes/SdesNotificationItem.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsrouter.models.sdes
+
+import play.api.libs.json._
+import uk.gov.hmrc.transitmovementsrouter.utils.RouterHeaderNames
+
+import java.time.Instant
+
+case class SdesNotificationItem(
+  notification: SdesNotification,
+  filename: String,
+  correlationID: String,
+  checksumAlgorithm: String,
+  checksum: String,
+  availableUntil: Instant,
+  failureReason: Option[String],
+  dateTime: Instant,
+  properties: Seq[SdesProperties]
+) {
+
+  val conversationId = properties
+    .find(
+      p => p.name.toLowerCase() == RouterHeaderNames.CONVERSATION_ID.toLowerCase()
```

Comment:
I think here we can remove ```toLowerCase()``` from both the side  and find with actual match

Reply:
As a note, we don't know exactly what the property name is but it _will_ be case sensitive given it'll be part of a Json schema. So, for that reason alone, we should remove the `toLowerCase`.

We just don't know what that property will be yet.